pyproject.toml

```diff
@@ -63,8 +63,6 @@ module = [
     "pytorch_lightning.core.mixins.hparams_mixin",
```

Comment:
you can delete the comment in line 39 "# Changes mypy default to ignore all errors" as it is no longer relevant